### ISSUE TITLE
UIBarButtonItemStyleBordered deprecated warning

### ISSUE THREAD

**Issue by swordray** [mariohahn/MHVideoPhotoGallery#99]
XCode 6.3.2

```
Pods/MHVideoPhotoGallery/MHVideoPhotoGallery/MMHVideoPhotoGallery/MHGalleryController/MHGalleryController.m:82:79: 'UIBarButtonItemStyleBordered' is deprecated: first deprecated in iOS 8.0 - Use UIBarButtonItemStylePlain when minimum deployment target is iOS7 or later
```

**Comment by marcelmika** [comment]
+1

**Comment by swordray** [comment]
Seems to be solved by #130.
